app/services/bulk_api_import/fhir_importable.rb

```diff
@@ -26,12 +26,4 @@ def translate_facility_id(id, org_id:)
         facility_groups: {organization_id: org_id})
       .facility.id
   end
-
-  def translate_patient_id(id, org_id:)
-    translate_id(
-      id,
-      org_id: org_id,
-      ns_prefix: "patient_business_identifier"
-    )
-  end
```

Comment:
This method ended up having only one usage, during patient creation, so I inlined it there and removed it from this mixin.